lib/perl/Genome/Model/ClinSeq/Command/Converge/Base.pm

```diff
@@ -36,6 +36,11 @@ class Genome::Model::ClinSeq::Command::Converge::Base {
               doc => 'minimum base quality of bases in reads to be considered',
               default => '20',
         },
+        force => { 
```

Comment:
I would encourage using a descriptive name for what this boolean allows (e.g., `--allow-multiple-individuals`) instead of a generic `--force`.